generators/client/templates/angular/src/test/javascript/spec/app/admin/audits/_audits.component.spec.ts

```diff
@@ -95,9 +95,9 @@ describe('Component Tests', () => {
                expect(comp.toDate).toBe(getDate());
                expect(comp.fromDate).toBe(getDate(false));
                expect(comp.itemsPerPage).toBe(ITEMS_PER_PAGE);
-               expect(comp.page).toBe(1);
+               expect(comp.page).toBe(10);
                expect(comp.reverse).toBeFalsy();
-               expect(comp.orderProp).toBe('timestamp');
+               expect(comp.predicate).toBe('id');
```

Comment:
it has to be `timestamp` right ?

Reply:
No I think it has to be `id` because it is set by mock router (see https://github.com/jhipster/generator-jhipster/blob/master/generators/client/templates/angular/src/test/javascript/spec/helpers/_mock-route.service.ts#L33).

Nonetheless, I think you right, `timestamp` should be the default sort, see my last commit correcting this.